123_state_tasks.R

```diff
@@ -30,8 +49,8 @@ do_state_tasks <- function(oldest_active_sites, ...) {
     task_plan = task_plan,
     makefile = '123_state_tasks.yml',
     include = 'remake.yml',
-    packages = c('tidyverse', 'dataRetrieval'),
-    sources = "'1_fetch/src/get_site_data.R'",
+    packages = c('tidyverse', 'dataRetrieval' , 'lubridate'),
+    sources = c('1_fetch/src/get_site_data.R', '3_visualize/src/plot_site_data.R',  '2_process/src/tally_site_obs.R', 'remake.yml'),
```

Comment:
I was expecting to see this line change by the time you submitted this PR. It's that last check box in the last open issue comment:

> Does this adding of source files feel familiar? You did it for 1_fetch/src/get_site_data.R a few issues ago, but it's an easy step to forget. You can actually save yourself the headache altogether if you set up your code a bit differently. First, see how the contents of ... in do_state_tasks() exactly match the contents of the sources argument to create_task_makefile()? This will be a consistent pattern, so rewrite that create_task_makefile() argument to use the ..., like so: sources = c(...). Now you only need to remember to add new source files for 123_state_tasks.yml in one place - remake.yml - so it should be easier to get right.

Reply:
Also, `remake.yml` isn't needed in this list.

Reply:
@aappling-usgs  Is the last check box issue resolved?

Reply:
No...